### PR TITLE
Fix Playwright E2E tests documentation as some steps to set up the local environment and run E2E tests were not accurate

### LINKED ISSUE #36867: The instructions for running Playwright E2E tests are missing the actual command to run these tests in the docs
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

There is a mistake in the Playwright E2E tests documentation where the instructions to run them are explained but the actual command to run the tests is missing, leading to confusion:

Screenshot 2023-02-17 at 12 22 56

### Expected behavior

The instructions for running the tests for the first time should be something like:

```
nvm use
pnpm install
pnpm run build --filter=woocommerce
pnpm env:start
pnpm test:e2e-pw
```

### Actual behavior

Instead, it's missing the actual command for running the tests, looking like this:

```
nvm use
pnpm install
pnpm run build --filter=woocommerce
pnpm env:start
```

This doesn't run the tests, just builds WooCommerce and sets up the local environment.

### Steps to reproduce

1. Head to the [Playwright E2E tests documentation](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e-pw/README.md).
2. Open a terminal and execute the steps described in the instructions for **running the tests for the first time**, inside the [Introduction](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e-pw/README.md#introduction) section:
```
nvm use
pnpm install
pnpm run build --filter=woocommerce
pnpm env:start
```

### WordPress Environment

This is a bug in the documentation, there is no need to share WordPress details.

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by alopezari**
### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

The Playwright E2E tests documentation was not accurate as it exposed obsolete yet working commands for running tests but also it was missing some important steps in the process that were causing confusion to new contributors. This PR aims to solve this by providing some light an accuracy in the steps to be performed to run E2E tests.

Closes [#36867](https://github.com/woocommerce/woocommerce/issues/36867).

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Review the changes and check they make sense.
2. Run the steps for `running tests for the first time:
-   `nvm use` (uses the default node version you have set in NVM)
-   `pnpm install` (installs dependencies)
-   `pnpm run build --filter=woocommerce` (builds WooCommerce locally)
-   `cd plugins/woocommerce` (heads to the WooCommerce plugin folder)
-   `npx playwright install` (makes sure to use the latest Playwright version)
-   `pnpm env:start` (starts the local environment)
-   `pnpm test:e2e-pw` (runs tests in headless mode)
3. Check the flow worked and you could run tests successfully in headless mode.
4. Try to run the tests again by restarting the environment:
-   `pnpm env:restart` (restarts the local environment)
-   `pnpm test:e2e-pw` (runs tests in headless mode)
5. Check that the `other ways of running tests` work:
-   `pnpm test:e2e-pw` (usual, headless run)
-   `pnpm test:e2e-pw --headed` (headed -- displaying browser window and test interactions)
-   `pnpm test:e2e-pw --debug` (runs tests in debug mode)
-   `pnpm test:e2e-pw ./tests/e2e-pw/tests/activate-and-setup/basic-setup.spec.js` (runs a single test)

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/README.md [hunk 1]
```diff
@@ -34,24 +34,27 @@ End-to-end tests are powered by Playwright. The test site is spinned up using `w
...
 **Running tests for the first time:**
 
--   `nvm use`
--   `pnpm install`
--   `pnpm run build --filter=woocommerce`
--   `pnpm env:start`
+-   `nvm use` (uses the default node version you have set in NVM)
+-   `pnpm install` (installs dependencies)
+-   `pnpm run build --filter=woocommerce` (builds WooCommerce locally)
+-   `cd plugins/woocommerce` (heads to the WooCommerce plugin folder)
```
Comment (lsinger) [inline plugins/woocommerce/tests/e2e-pw/README.md]:
```suggestion
-   `cd plugins/woocommerce` (change into the WooCommerce plugin folder)
```

Might be a bit more comprehensible for non-native speakers.

File: plugins/woocommerce/tests/e2e-pw/README.md [hunk 2]
```diff
@@ -34,24 +34,27 @@ End-to-end tests are powered by Playwright. The test site is spinned up using `w
...
+-   `npx playwright install` (makes sure to use the latest Playwright version)
```
Comment (lsinger) [inline plugins/woocommerce/tests/e2e-pw/README.md]:
```suggestion
-   `npx playwright install` (install the latest Playwright version)
```

**Review by lsinger — DISMISSED** [review]
LGTM! 🚢

**Review by nigeljamesstevenson — APPROVED** [review]
thanks a lot for these updates!

**Pull request merged**